s3/src/main/scala/quasar/destination/avalanche/s3/AvalancheS3DestinationModule.scala

```diff
@@ -110,39 +106,40 @@ object AvalancheS3DestinationModule extends AvalancheDestinationModule[Avalanche
     init.value
   }
 
+  // skip validation for now, see ch11655
   private def validBucket[F[_]: Concurrent: ContextShift](
       client: S3AsyncClient,
       sanitizedConfig: Json,
       bucket: Bucket)
-      : F[Either[InitError, Unit]] =
-    S3StatusService(client, bucket) map {
-      case BlobstoreStatus.Ok =>
-        ().asRight
-
-      case BlobstoreStatus.NotFound =>
-        DestinationError
-          .invalidConfiguration[Json, InitError](
-            destinationType,
-            sanitizedConfig,
-            ZNel("Upload bucket does not exist"))
-          .asLeft
-
-      case BlobstoreStatus.NoAccess =>
-        DestinationError
-          .accessDenied[Json, InitError](
-            destinationType,
-            sanitizedConfig,
-            "Access denied to upload bucket")
-          .asLeft
-
-      case BlobstoreStatus.NotOk(msg) =>
-        DestinationError
-          .invalidConfiguration[Json, InitError](
-            destinationType,
-            sanitizedConfig,
-            ZNel(msg))
-          .asLeft
-    }
+      : F[Either[InitError, Unit]] = ().asRight[InitError].pure[F]
```

Comment:
Might be a bit more direct to comment out the call-site than the impl.

Reply:
Yes that's true but the disadvantage is that then there's a warning `private method validBucket is never used`, so we would also need to comment out that `private` to get rid of that warning. 

And because we need to actively think about _removing_ that comment to make it private again when we reinstate the call to `validBucket` I prefer the change as it currently is.

An alternative that I'd also be happy with is to comment out both the call-site and the complete impl of `validBucket`.

Reply:
Ah, bummer. I'd say just comment them both out then 👍